hknweb/exams/urls.py

```diff
@@ -1,9 +1,16 @@
 from django.conf.urls import url
 from django.urls import path
+from .exam_autocomplete import CourseSemesterAutocomplete
 
 from . import views
 
 urlpatterns = [
 	path('course/<department>/<number>/', views.exams_for_course),
+    path('new', views.add_exam),
     url(r'^$', views.index),
+    url(
+        r'^courseSemester-autocomplete/$',
```

Comment:
Any reason we're using `url` (which is [deprecated](https://docs.djangoproject.com/en/3.0/ref/urls/#url)) and a regex for a flat string match? I'd prefer we use `path`, and `re_path` only when necessary.

Reply:
Fixed